### DISCUSSION TITLE
Install Custom Detection Rules

### DISCUSSION THREAD

**Discussion by earthbeaker** [Security-Onion-Solutions/securityonion#2649] [Unsupported Versions] [answered]
Hi Folks,

Can we install custom Elastic SIEM rules into Security Onion 

https://github.com/elastic/detection-rules
https://github.com/elastic/detection-rules/tree/main/rules
https://my.socprime.com/integrations/

If yes, then how.Please guide.

Thanks

**Comment by TOoSmOotH** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/local-rules.html?highlight=rules

https://docs.securityonion.net/en/2.3/elastalert.html?highlight=elastalert
